src/odemis/acq/stream/_base.py

```diff
@@ -886,7 +886,6 @@ def _updateDRange(self, data=None):
                                 mn_p2 = 0
                             elif mn > 0:
                                 mn_p2 = 2 ** int(math.floor(math.log(mn + 1, 2)))
```

Comment:
This still causes some issues... when there is only one value, which is a multiple of 256 (mn > mx).